sts.md: grammar/typos

trobotham opened the pull request.

SaberUK commented on extensions/sts.md:
```diff
@@ -174,7 +174,7 @@ Constant values into the future can be achieved by a configured number of second
 Fixed expiry times will involve a dynamic `duration` value being calculated on each connection attempt.
 
-Server implementers should be aware that fixed expiry times might not be precisely guaranteed in the case where clients reschedule policy expiry on disconnect, or periodically while connected.
+Server implementors should be aware that fixed expiry times might not be precisely guaranteed in the case where clients reschedule policy expiry on disconnect, or periodically while connected.
```
This was correct previously.

dgw commented on extensions/sts.md:
```diff
@@ -84,7 +84,7 @@ Servers MAY send this key to all clients, but insecurely connected clients MUST
 Preload list providers MUST only consider hosts for inclusion after validating their connection security and ensuring a valid STS policy with a `preload` key is in place. This allows IRC network administrators to opt-in for inclusion in preload lists.
 
-Servers SHOULD be prepared to offer secure connections for the long term when enabling a preload policy. Timely removal of hostnames from preload lists might not be possible.
+Servers SHOULD be prepared to offer secure connections for the long-term when enabling a preload policy. Timely removal of hostnames from preload lists might not be possible.
```
This is correct without the hyphen.

trobotham replied:
I have to disagree on this but I will change it now.

dgw replied:
The hyphen would make this an incomplete phrase. "Long-term" is an adjective, while "long term" is a noun ("term") described by an adjective ("long").

This particular phrase is a fairly common mix-up in writing, but the presence (or absence) of a hyphen is important.

DanielOaks commented:
Eh, cool. +1 for a -> an

jwheare commented:
a -> an is well spotted thanks, instantly mergeable. delete -> remove seems like a pedantic change for the sake of change thing i don't really wanna have a long discussion on. Take that bit out and I'll merge straight away.

trobotham commented:
> a -> an is well spotted thanks, instantly mergeable. delete -> remove seems like a pedantic change for the sake of change thing i don't really wanna have a long discussion on. Take that bit out and I'll merge straight away.

I changed delete to remove to keep consistency with the rest of the document, reverted.

jwheare commented:
Consistency is good, I unreverted it. In future, good things to state up front in a PR description.
